### ISSUE TITLE
Multiple components, multiple files, same file name: how to distinguish?

### ISSUE THREAD

**Issue by KLynn2019** [asciidoctor/asciidoctor-intellij-plugin#1309]
I'm not sure how common my situation is, which is why I'm listing it as a feature request instead of a bug...

As project administrator, I have all of my team's Antora components (40+) cloned on my machine. Currently they are grouped in different IntelliJ projects depending on the types of sites they create.

While creating a navigation file, I encountered this situation:

![image](https://user-images.githubusercontent.com/51419423/223452118-82c67b54-2250-4e6f-844b-454e44ac5e21.png)

Typically I'm able to see the component associated with a particular file but that didn't happen in this case and I'm thinking it's because there are so many index.adoc files in this particular project.

I decided to choose the first one. The preview looked like this:

![image](https://user-images.githubusercontent.com/51419423/223453083-28e0a42e-c1ce-4c60-8373-b28122a1ff74.png)

Obviously a concern...but when I ran Antora on this component, this is the output I got:

![image](https://user-images.githubusercontent.com/51419423/223453321-ebcad801-9223-4f7d-a57f-b0bbada821c4.png)

Here's my question/possible suggestion: When creating an xref, image, include, etc., shouldn't the associated component display anyway regardless of the number of files in a project that happen to have the same name?

**Comment by ahus1** [comment]
Hi @KLynn2019 - thanks for sharing this. I see that this needs some improvement. 

I tried to replicate it locally, and here's the way I was able to come up with two `index.adoc` in my auto-complete: 

* Create a new project with a layout like this: 
   ![image](https://user-images.githubusercontent.com/3957921/223513596-b88a5563-75d1-4677-980c-e6c36188166f.png)
* Both components need to have the same name and version in their component descriptor `antora.yml`

When I have a `nav.adoc` in `modules/ROOT`, the auto-complete will only pick up the files from the same component name and version. So I would assume you've checked out the components multiple times and opened it as one big project? This might have happend when you grouped them as described above. 

If this is the case: This will stop from happening if you would open each group as a project, and not all groups together as a big project. 

If this is not the case, I ask you to create a minimal ZIP file with an example project that shows at least two `index.adoc` on auto-complete.
